<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>1021</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="login_page">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>1021</width>
        <height>581</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="login_frame">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>200</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="user_label">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>50</y>
            <width>49</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>User</string>
          </property>
         </widget>
         <widget class="QLabel" name="password_label">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>100</y>
            <width>49</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="password_input">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>120</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>****</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="user_input">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>70</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
          <property name="placeholderText">
           <string>root</string>
          </property>
         </widget>
         <widget class="QPushButton" name="login_button">
          <property name="geometry">
           <rect>
            <x>470</x>
            <y>160</y>
            <width>75</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="main_page">
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>1021</width>
        <height>581</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Alunos</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Avaliações</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Disciplinas</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Professores</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <widget class="QPushButton" name="update_button">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>540</y>
            <width>75</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
         <widget class="QPushButton" name="delete_button">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>540</y>
            <width>75</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="content_search_field">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>191</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>271</width>
            <height>191</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="vertical_fields"/>
         </widget>
         <widget class="QPushButton" name="search_button">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>10</y>
            <width>51</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
         <widget class="QPushButton" name="add_button">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>250</y>
            <width>75</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>user_input</tabstop>
  <tabstop>password_input</tabstop>
  <tabstop>login_button</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>update_button</tabstop>
  <tabstop>delete_button</tabstop>
  <tabstop>content_search_field</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
